static/js/components/VideoPlayer.js

```diff
@@ -129,38 +145,66 @@ export default class VideoPlayer extends React.Component {
     this.videoNode.style.left = `${left}px`;
     this.videoNode.style.top = `${top}px`;
     // $FlowFixMe prop does not have to be a number
-    this.videoNode.style[prop] = 'scale(2)';
+    this.videoNode.style[prop] = "scale(2)";
     this.configureCameras();
 
   }
 
   toggleFullscreen() {
-    if (isFullscreen()) {
+    let fullscreen = isFullscreen();
+    if (fullscreen) {
       // $FlowFixMe
       document[FULLSCREEN_API.exitFullscreen]();
     } else {
       // $FlowFixMe videoContainer.parentElement is not going to be null
       this.videoContainer.parentElement[FULLSCREEN_API.requestFullscreen]();
     }
+    this.player.el_.dispatchEvent(new Event(`fullscreen ${fullscreen ? "off" : "on"}`));
+  }
+
+  analyzeEvent(action: string, label: string) {
```

Comment:
this function is kinda mis-named to me, since what we're doing here is not analyzing an event but setting up an event handler which will do that when the event fires. Can we call it something like `createEventHandler` instead?

Reply:
sure